cmd/plugins/memory-policy/Dockerfile

```diff
@@ -0,0 +1,28 @@
+ARG GO_VERSION=1.23
+
+FROM golang:${GO_VERSION}-bullseye AS builder
+
+ARG IMAGE_VERSION
+ARG BUILD_VERSION
+ARG BUILD_BUILDID
+WORKDIR /go/builder
+
+# Fetch go dependencies in a separate layer for caching
+COPY go.mod go.sum ./
+COPY pkg/topology/ pkg/topology/
+RUN go mod download
+
+# Build nri-resmgr
+COPY . .
+
+RUN make clean
+RUN make IMAGE_VERSION=${IMAGE_VERSION} BUILD_VERSION=${BUILD_VERSION} BUILD_BUILDID=${BUILD_BUILDID} PLUGINS=nri-memory-policy build-plugins-static
+RUN make -C cmd/plugins/memory-policy/mpolset
```

Comment:
So far I think we tried to keep a single monolithic Makefile for all plugins, so I wonder if this extra Makefile is really necessary. I'm guessing that you want to force compiling to a static binary this way and maybe it is the most straightforward one, but I wonder if that could be done in the top level Makefile for this plugin somehow using the existing related plumbing... anyway, not a blocker.